contentctl/objects/abstract_security_content_objects/security_content_object_abstract.py

```diff
@@ -1,31 +1,39 @@
 from __future__ import annotations
-from typing import TYPE_CHECKING, Self
+from typing import TYPE_CHECKING, Self, Any
 
+# TODO (cmcginley): this config object does not seem to exist and is not used anywhere in this
+#   file; can we remove?
 if TYPE_CHECKING:
-    from contentctl.objects.deployment import Deployment        
+    from contentctl.objects.deployment import Deployment
     from contentctl.objects.security_content_object import SecurityContentObject
     from contentctl.objects.config import Config
```

Comment:
Can we remove this `Config`? It's not used anywhere and does not seem to exist.

Additionally, what's happening here w/ the check against `TYPE_CHECKING`?

Reply:
Yes, this has now been removed.